lib/api/apiUtils/object/objectRestore.js

```diff
@@ -0,0 +1,146 @@
+const async = require('async');
+const { errors, s3middleware } = require('arsenal');
+
+const { allowedRestoreObjectRequestTierValues } = require('../../../../constants');
+const coldStorage = require('./coldStorage');
+const monitoring = require('../../../utilities/monitoringHandler');
+const { pushMetric } = require('../../../utapi/utilities');
+const { parseRestoreRequestXml } = s3middleware.objectRestore;
+
+const METHOD = 'objectRestore';
```

Comment:
I would suggest putting this const in the `objectRestore` function, no need to have it as a global variable.